### ISSUE TITLE
UN5.2C-TINY

### ISSUE THREAD

**Issue by JHS1211** [Erikovsky/UN5.2C-TINY#1]
Hello, first of all, thank you for your work and share, very beautiful, I like this one. I have a question to consult you that power supply PCB board in the Q7 and schematic diagram of the Q7 connection is not the same, in the power supply PCB board is Q7 B - E connect GND, the Q7 and schematic diagram is very connect GND B, C, I am not very understand, this is a better way to work? I like this small plate welding machine, hope to get your help, I thank you very much. My English is not good, please forgive me.
![001](https://user-images.githubusercontent.com/67097777/126430147-1177f288-a867-48c0-abbf-08931ad976cd.png)

**Comment by JHS1211** [comment]
Hello, first of all, thank you for your work and share, very beautiful, I like this one. I have a question to consult you that power supply PCB board in the Q7 and schematic diagram of the Q7 connection is not the same, in the power supply PCB board is Q7 B - E connect GND, the Q7 and schematic diagram is very connect GND B, C, I am not very understand, this is a better way to work? I like this small plate welding machine, hope to get your help, I thank you very much. My English is not good, please forgive me.

**Comment by Erikovsky** [comment]
You are right, the pinout on schematic is B-E-C, which is wrong. There are 2 versions of BCX56-16, the SOT223 and SOT89.
Keep in mind that SOT223 and SOT89 have differed pinout.
SOT223: B-C-E
SOT89: E-C-B
While B and C are shorted on the schematic, so it would either be 1-2 or 2-3 shorted to GND.

**Comment by JHS1211** [comment]
无论是223或89的封装，只是管脚标注的号相反，但在实际的元件面前其实是一样的，正面看，左侧的第一脚都是B，中间脚都是C，右侧脚都是E，只是标注的编号不同，实际应用没区别的。谢谢你的回复。


发自我的iPhone

------------------ Original ------------------
From: 甘草酸不酸 ***@***.***&gt;
Date: Sun,Aug 15,2021 2:35 AM
To: Erikovsky/UN5.2C-TINY ***@***.***&gt;
Cc: JHS1211 ***@***.***&gt;, Author ***@***.***&gt;
Subject: Re: [Erikovsky/UN5.2C-TINY] UN5.2C-TINY (#1)





 
You are right, the pinout on schematic is B-E-C, which is wrong. There are 2 versions of BCX56-16, the SOT223 and SOT89.
 Keep in mind that SOT223 and SOT89 have differed pinout.
 SOT223: B-C-E
 SOT89: E-C-B
 While B and C are shorted on the schematic, so it would either be 1-2 or 2-3 shorted to GND.
 
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by JHS1211** [comment]
Either 223 or 89 of encapsulation, just pin marks, on the contrary, but before the actual component is the same, positive, to the left of the first foot is B, middle C, on the right foot is E, just indicate the number of different, practical application of no difference. Thank you for your reply.

**Comment by heuboda** [comment]
Hello, what a nice project! Thank you for your work. May I ask if it is possible to use the JBC T470 which uses 48V? At the schematic I haven't found any thing, but at your picture there is 48V. 
Is it possible to use the sparkybg ´s software, or do I have to use your firmware? 
Thank you for your help.

**Comment by Erikovsky** [comment]
The PCB is just re-drawed one, so nothing is changed; it's just another shape of Sparky's design, so it's compatible with Sparky's firmware, mantainance software.
